SeleniumTestXUnit/Tests/Features/Item/CheckItem.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Check an Item
+As a logged user, the user wants to check an item
+So that the user can keep track of the items that are done
+
+
+Background:
+    Given the user is logged in
+        And the user has an existing project
+        And the project has one or more existing items
```

Comment:
Try to be more specific about how many items the project should have, anyway this step should be a BeforeScenario hook instead.